x-pack/docs/en/security/securing-communications/update-tls-certificates.asciidoc

```diff
@@ -0,0 +1,529 @@
+to sign the certificate that is currently in use. It also assumes that the
+clients connecting to {es} on the HTTP layer are configured to trust the CA
+certificate.
+
+TIP: If your CA has changed, complete the steps in 
+<<update-node-certs-different,update security certificates with a different CA>>.
+
+[[node-certs-same-transport]]
+==== Generate a new certificate for the transport layer
+Create a new PKCS#12 file that contains a certificate and key and replace
+the existing PKCS#12 file (such as `elastic-certificates.p12`) with the new
+certificate. 
+
+NOTE: The following examples use PKCS#12 files, but the same steps apply to JKS keystores.
+
+. Open the `ES_PATH_CONF/elasticsearch.yml` file and check the names of the
+security certificates that are currently in use. You'll use the same names
+for your new certificate.
++
+In this example, the keystore and truststore are pointing to different files.
+Your configuration might use the same file that contains the certificate and CA.
++
+[source,yaml]
+----
+xpack.security.transport.ssl.keystore.path: config/elastic-certificates.p12
+xpack.security.transport.ssl.keystore.type: PKCS12
+xpack.security.transport.ssl.keystore.password: password
+xpack.security.transport.ssl.truststore.path: config/elastic-ca.p12
+xpack.security.transport.ssl.truststore.type: PKCS12
+xpack.security.transport.ssl.truststore.password: password
+----
+
+. On any node in your cluster, generate a certificate and private key for your
+node. You must include the CA that was used to sign the certificate currently
+in use.
++
+[source,shell]
+----
+./bin/elasticsearch-certutil cert --ca elastic-stack-ca.p12
+----
++
+[%collapsible%open]
+.Command parameters
+====
+   `--ca <ca_file>`:: Name of the CA file used to sign your certificates. The
+   default file name from the `elasticsearch-certutil` tool is
+   `elastic-stack-ca.p12`.
+====
++
+    a. Enter the password for your CA when prompted.
+
+    b. Enter the name for your certificate, which must match the name of the
+    certificate currently in use.
+
+    c. (Optional) Enter a password for your certificate. 
+
+. If you entered a password when creating the node certificate, run the following commands to store the password in the {es} keystore:
++
+--
+[source,shell]
+----
+./bin/elasticsearch-keystore add xpack.security.transport.ssl.keystore.secure_password
+----
+
+[source,shell]
+----
+./bin/elasticsearch-keystore add xpack.security.transport.ssl.truststore.secure_password
+----
+--
+
+. [[cert-password-updates]] If your certificate is password protected, the password is stored in the {es} keystore, _and_ the password needs to change, then you must perform a <<restart-cluster-rolling,rolling restart>> on your cluster. Complete the steps until the step that indicates *Perform any needed changes*, and then continue to the next step to replace your existing certificates.
+
+. Replace your existing certificate with the new certificate, ensuring that the
```

Comment:
certificate -> keystore . Maybe also name it to be precise ?

Reply:
++ I'll change to read:

>Replace your existing keystore with the new keystore, ensuring that the file names match. For example, `elastic-stack-ca.p12`.